examples/teleop-peer/package.json

```diff
@@ -7,10 +7,14 @@
     "serve": "vite preview"
   },
   "devDependencies": {
-    "@formant/data-sdk": "0.0.87",
+    "@formant/data-sdk": "^1.18.0",
+    "@formant/realtime-sdk": "^1.2.0",
     "@formant/ui-sdk-joystick": "latest",
     "@formant/ui-sdk-realtime-player": "latest",
     "typescript": "^4.3.2",
     "vite": "^2.5.4"
+  },
+  "dependencies": {
+    "grpc-web": "^1.5.0"
```

Comment:
Why does this need to be added?

Reply:
`yarn run build` will fail without it

Reply:
<img width="1137" alt="image" src="https://github.com/FormantIO/toolkit/assets/35079898/354fd23e-ea9d-4340-94b9-faadd9ec737f">

Reply:
Oh, yarn isn't supported in this repo. Everything breaks if you don't use npm.

Reply:
ok, that's really confusing:
- there is no yarn.lock or package-lock.json in the repo
- [The documentation says explicitly to use yarn](https://github.com/FormantIO/toolkit/blob/master/examples/teleop-peer/README.md):
<img width="470" alt="image" src="https://github.com/FormantIO/toolkit/assets/35079898/f1987313-6c20-4647-86ef-9cb9c45dddbd">

Reply:
If you use `npm run build`:
You get: 

<img width="1142" alt="image" src="https://github.com/FormantIO/toolkit/assets/35079898/0ecf83f2-3ab2-4354-9da1-e497bb1ad244">